utils/docker-ce/files/dockerd.init

```diff
@@ -43,10 +43,39 @@ process_config() {
 	json_dump > "$DOCKERD_CONF"
 }
 
+process_ac_allowed() {
+	local cfg="$1"
+
+	local iface
+
+	config_get iface "$cfg" iface
```

Comment:
Do we want to name this option `allowed_iface`?
Also, We may want to add a commented out example of this to the default config file.